source/bringauto/modules/StatusAggregator.cpp

```diff
@@ -27,7 +27,9 @@ StatusAggregator::aggregateStatus(structures::StatusAggregatorDeviceState &devic
 								  const unsigned int &device_type) {
 	auto &currStatus = deviceState.getStatus();
 	Buffer aggregatedStatusBuff {};
-	module_->aggregateStatus(aggregatedStatusBuff, currStatus, status, device_type);
+	if (module_->aggregateStatus(aggregatedStatusBuff, currStatus, status, device_type) != OK) {
```

Comment:
if aggregate status fails it sends an empty status to the external server - is this behaviour correct?

Reply:
aggregateStatus will now return the current status if the module function fails